internal/tiltfile/value/path.go

```diff
@@ -24,3 +26,53 @@ func (p *LocalPath) Unpack(v starlark.Value) error {
 	p.Value = str
 	return nil
 }
+
+type LocalPathList struct {
+	t                  *starlark.Thread
+	Value              []string
+	AllowsSingleString bool
+}
+
+func NewLocalPathListUnpacker(t *starlark.Thread) LocalPathList {
+	return LocalPathList{
+		t: t,
+	}
+}
+
+func (p *LocalPathList) Unpack(v starlark.Value) error {
+	if p.AllowsSingleString {
+		_, ok := AsString(v)
+		if ok {
+			str, err := ValueToAbsPath(p.t, v)
+			if err != nil {
+				return err
+			}
+			p.Value = []string{str}
+			return nil
+		}
+	}
+
+	var iter starlark.Iterator
+	switch x := v.(type) {
+	case *starlark.List:
+		iter = x.Iterate()
+	case starlark.Tuple:
+		iter = x.Iterate()
+	default:
+		return fmt.Errorf("value should be a string or List or Tuple of strings, but is of type %s", v.Type())
+	}
+
+	defer iter.Done()
+
+	values := []string{}
+	var item starlark.Value
+	for iter.Next(&item) {
+		str, err := ValueToAbsPath(p.t, item)
+		if err != nil {
+			return fmt.Errorf("unpacking list item #%d: %v", len(values), err)
```

Comment:
```suggestion
			return fmt.Errorf("unpacking list item at index %d: %v", len(values), err)
```

Reply:
maybe just me,  but I think "index" is more unambiguously 0-based - without other information, I'd bias towards interpreting "#%d" as 1-based

And/or putting the value of `item.String()` in the error message would make clear which element is problematic. (though that might be better done inside of `ValueToAbsPath`)

Reply:
i agree!